test/popcorn/eval_test.exs

```diff
@@ -672,7 +672,10 @@ defmodule Popcorn.EvalTest do
       |> AtomVM.try_eval(:elixir, run_dir: dir)
 
     assert %{exit_status: 0, output: output} = result
-    assert [debug, info, warning, error] = String.split(output, "\n", trim: true)
+
+    assert [debug, info, warning, error] =
+             String.split(output, "\n", trim: true) |> Enum.reject(&String.starts_with?(&1, "\e"))
```

Comment:
What's the `\e`?

Reply:
For some reason (#105 ?) logger started coloring its output, and there are some lines in form of \e***\n, that indicate start/end of particular color